### PR TITLE
Clean command to purge bot messages from channel.

### PR DESCRIPTION
**Pull request by Saud-97**
**Please describe the changes this PR makes and why it should be merged:**
Added a clean command to delete bot messages from channel.

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

File: commands/slash/clean.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            botMessages.shift();
+            interaction.channel.bulkDelete(botMessages, true)
+                .then(async deletedMessages => {
+                    //Filtering out messages that did not get deleted.
+                    messages = messages.filter(msg => {
+                        !deletedMessages.some(deletedMsg => deletedMsg == msg);
+                    });
+                    if (messages.size > 0) {
+                        client.log(`Deleting [${messages.size}] messages older than 14 days.`)
+                        messages.deleteAll();
```
Comment (Saud-97) [inline commands/slash/clean.js:34]: I need someone to test deletion of old bot messages older than 14 days. I couldn't test for this edge case.

Comment (XstreamSpeed) [reply commands/slash/clean.js:34]: I'll test it it later, but like why a clean command tho it's it a music bot.. I have tons of my own commands top tho here is the list of commands that I made extra in discord music bot:

Bugreport Feedback 8ball
Wordle Minesweeper Eval
Execute Configinfo Banner Instagram Setstatus
And weather and they are all tested and work 100% hit me up on discord if you're interested in any of those files- Mex#1222

Comment (Neko-Life) [reply commands/slash/clean.js:35]: > I need someone to test deletion of old bot messages older than 14 days

I don't think you can delete messages older than 2 weeks, it's discord limitation.

**Comment by Saud-97** [comment]
You are talking about bulk delete. Notice I'm not using it for old messages
deletion I'm deleting them one by one

**Comment by Saud-97** [comment]
I have changed the deletion of old messages and made it work and was tested. This is now ready to merge.

**Review by DarrenOfficial — APPROVED** [review]
